WordPress/Classes/ViewRelated/Stats/Period Stats/Overview/OverviewCell.swift

```diff
@@ -97,4 +131,26 @@ private extension OverviewCell {
         differenceLabel.text = tabData.differenceLabel
         differenceLabel.textColor = tabData.differenceTextColor
     }
+
+    // MARK: Chart support
+
+    func resetChartView() {
+        for subview in chartContainerView.subviews {
+            subview.removeFromSuperview()
+        }
+    }
+
+    func configureChartViewIfNeeded() {
```

Comment:
I'm a little confused by this method name, specifically the `IfNeeded` part. Is the chart ever _not_ needed on the Overview? There isn't a condition within the the method that indicates that scenario.

Reply:
This is valid feedback. I've pushed a commit ([`3779c0a`](https://github.com/wordpress-mobile/WordPress-iOS/pull/11210/commits/3779c0a6d9e499c94047fb9fa85bb9449b2c3073)) that removes the `IfNeeded` suffix.